scripts/lib/libvirt/compute-config

```diff
@@ -34,7 +35,10 @@ def main():
 
     args = parser.parse_args()
 
-    print(libvirt_setup.compute_config(args))
+    machine="pc-0.14"
+    if os.system("rpm -q qemu|grep -q qemu-2")==0:
```

Comment:
Uhm, this is interesting. I bet that we are going to use a lot more rpm. I suggest to use this:

https://docs.fedoraproject.org/ro/Fedora_Draft_Documentation/0.1/html/RPM_Guide/ch16s03s02.html